tests/src/test/java/alluxio/client/fuse/AbstractFuseIntegrationTest.java

```diff
@@ -204,6 +204,15 @@ public void cp() throws Exception {
   public void ddDuAndRm() throws Exception {
     String testFile = "/ddTestFile";
     createFileInFuse(testFile);
+    AlluxioURI testFileUri = new AlluxioURI(testFile);
+
+    CommonUtils.waitFor("File write completed", () -> {
```

Comment:
shall we add time out here?

Reply:
please also add the reason why we need to wait here (due to async release).
Let's also create a ticket on this?